libs/brain/button/src/lib/brn-button.ts

```diff
@@ -0,0 +1,36 @@
+import { booleanAttribute, Directive, ElementRef, inject, input, NgZone, OnDestroy, Renderer2 } from '@angular/core';
+
+@Directive({
+	selector: 'a[brnButton], button[brnButton]',
+	host: {
+		'[attr.tabindex]': 'disabled() ? -1 : undefined',
+		'[attr.disabled]': 'disabled() || null',
```

Comment:
The disabled attribute is not valid on a link only a button, so we should check to see if this is a button and only apply it then.